app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -34,6 +34,14 @@ def environment_url
       end
     end
 
+    def embed_js
+      case BoltConfiguration.environments[environment]
+      when 0 then 'https://connect.bolt.com/embed.js'
+      when 1 then 'https://connect-sandbox.bolt.com/embed.js'
+      when 2 then 'https://connect-staging.bolt.com/embed.js'
```

Comment:
We don't have the staging option. https://docs.bolt.com/developers/guides/checkout-guides/embedded-checkout/accounts/setup/#embedjs

Reply:
https://github.com/nebulab/solidus_bolt/commit/971f2cfda14aa0e4856498e7fc576655cbe06583